website/src/react-luma/components/luma-example.tsx

```diff
@@ -63,52 +68,78 @@ export const LumaExample: FC<LumaExampleProps> = (props: LumaExampleProps) => {
   let containerName = 'ssr';
 
   /** Each example maintains an animation loop */
-  const [animationLoop, setAnimationLoop] = useState<AnimationLoop | null>(null);
+  const animationLoopRef = useRef<AnimationLoop | null>(null);
+  const [canvas, setCanvas] = useState<HTMLCanvasElement | null>(null);
+  const usedCanvases = useRef(new WeakMap())
```

Comment:
Nit: Can we type this weakmap?
```suggestion
  const usedCanvases = useRef(new WeakMap<HTMLCanvasElement>())
```